### PR TITLE
[SPARK-27075] Remove duplicate execution tag parameters from the url, when accessing the execution table in the SQL page

### PR DESCRIPTION
**Pull request by shahidki31**
## What changes were proposed in this pull request?

When we sort any columns in the execution table of the SQL page in the WEBUI, it throws IllegalArgumentException. The root cause is that,  in the url, we are duplicating the execution tag parameters in the 'parameterPath'. Actually we should filter out the executionTag related entries while getting the 'parameterOtherTable'
https://github.com/apache/spark/blob/e9e8bb33ef9ad785473ded168bc85867dad4ee70/sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala#L161-L163
https://github.com/apache/spark/blob/e9e8bb33ef9ad785473ded168bc85867dad4ee70/sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala#L241
https://github.com/apache/spark/blob/e9e8bb33ef9ad785473ded168bc85867dad4ee70/sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala#L263-L266

## How was this patch tested?
Manually tested
Test steps:
Sort any column in the sql page execution table
Before fix:
screenshot from 2019-03-07 01-38-17

After fix:
screenshot from 2019-03-07 02-01-40

### CODE REVIEW

**Comment by shahidki31** [comment]
Jenkins, test this please

**Comment by shahidki31** [comment]
cc @vanzin Kindly review the PR.

File: sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala [hunk 1]
```diff
@@ -158,7 +158,8 @@ private[ui] class AllExecutionsPage(parent: SQLTab) extends WebUIPage("") with L
     showSucceededJobs: Boolean,
     showFailedJobs: Boolean): Seq[Node] = {
 
-    val parameterOtherTable = request.getParameterMap().asScala.map { case (name, vals) =>
+    val parameterOtherTable = request.getParameterMap().asScala
+      .filterNot(_._1.startsWith(executionTag)).map { case (name, vals) =>
```
Comment (vanzin) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala]:
```
val something = blah
  .filterNot(...)
  .map { blah =>
     ...
  }
```

Comment (shahidki31) [reply sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala]: Done. Thanks

**Comment by vanzin** [comment]
Merging to master.

**Comment by shahidki31** [comment]
Thanks @vanzin

**Pull request closed without merging**